src/lib/module.html

```diff
@@ -70,6 +70,60 @@
    */
   define.amd = {};
 
+  // `define.withImports`
+
+  /**
+   * Define an AMD module in a HTML import, and reference any dependencies that
+   * are expressed via <link rel="import"> elements.
+   *
+   * This is a simple wrapper around `define` that reduces module boilerplate:
+   *
+   *   <link rel="import" href="./foo.html">
+   *   <link rel="import" href="../bar.html">
+   *   &lt;script&gt;
+   *     define(['./foo.html', '../bar.html'], function(foo, bar) {
+   *       // ...
+   *     });
+   *   &lt;/script&gt;
+   *
+   * To something a little more convenient:
+   *
+   *   <link rel="import" href="./foo.html">
+   *   <link rel="import" href="../bar.html">
+   *   &ltscript&gt;
+   *     define(function(foo, bar) {
+   *       // ...
+   *     });
+   *   &lt/script&gt;
+   *
+   * **All** import link elements will be declared as dependencies for the
+   * defined module, and they will be declared in the same order.

```

Comment:
what happens with html imports that don't export anything because they don't have a script? it'd be nice to get their `<dom-module>` or document...